async-nats/src/lib.rs

```diff
@@ -315,8 +315,11 @@ impl ConnectionHandler {
         ping_period: Duration,
         flush_period: Duration,
     ) -> ConnectionHandler {
-        let ping_interval = interval(ping_period);
-        let flush_interval = interval(flush_period);
+        let mut ping_interval = interval(ping_period);
+        ping_interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
```

Comment:
I think that `Delay` should be used for ping.

Reply:
It shouldn't be necessary to manually reset it